support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -271,6 +271,37 @@ export function ThreeTierLanding(): JSX.Element {
 		);
 	};
 
+	const getCardContentBaseObject = (cardNumber: 1 | 2 | 3) => {
+		return {
+			title: tierCards[`tier${cardNumber}`].title,
+			benefits: tierCards[`tier${cardNumber}`].benefits,
+			isRecommended: !!tierCards[`tier${cardNumber}`].isRecommended,
+			isUserSelected: isCardUserSelected(
+				tierCards[`tier${cardNumber}`].plans[regularProductTypeKey].charges[
+					countryGroupId
+				].price,
+				tierCards[`tier${cardNumber}`].plans[regularProductTypeKey].charges[
+					countryGroupId
+				].discount?.price,
+			),
+			planCost:
+				tierCards[`tier${cardNumber}`].plans[regularProductTypeKey].charges[
+					countryGroupId
+				],
+		};
+	};
+
+	const generateTopTierGWCheckoutLink = () => {
```

Comment:
Later on, perhaps we need to use this approach to send through all url params to the tier1, tier2 checkouts?

Reply:
> Later on, perhaps we need to use this approach to send through all url params to the tier1, tier2 checkouts?

The top tier will use and anchor tag with the url params appended .. whereas the bottom 2 tiers and the single contribution are all clientside routed therefore use the `navigateWithPageView` function ... so you'll have to amend the `pageView` part of that function to ensure the right stuff is tracked